views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -29,10 +29,10 @@ define([
     'i18n',
     'core/plugin',
     'ui/hider',
-    'taoItems/preview/preview',
+    'taoItems/previewer/factory',
     'tpl!taoQtiItem/qtiCreator/plugins/button',
     'util/url'
```

Comment:
`util/url` looks to be a redundant dependency now. You can remove it.

Reply:
U r right, done